serverless.yml

```diff
@@ -58,68 +58,54 @@ functions:
 #      - http: ANY /
 #      - http: 'ANY {proxy+}'
 
-  catmemes:
-    handler: lambda/handler.catMemes
-    name: 'hellocats-${env:REACT_APP_STAGE}'
+  publicDash:
```

Comment:
Can the names be standardized in terms of capitalization? Instead of being personal-Dash and Team-Dash